### PR TITLE
Make the ocf server send simulated data.

### PR DESCRIPTION
**Pull request by ichbinblau**
Hi, @kiendinh I added some code to mimic the sensor data for easier test. Plz review.

### CODE REVIEW

File: ocf-servers/js-servers/environmental_sensor.js [hunk 1]
```diff
@@ -18,17 +18,19 @@ var	device = require('iotivity-node')('server'),
...
-			uvIndex : 0};
+			uvIndex : 0},
+    simData = 0.0;
 
 var noble = '';
 try {
 	noble = require('noble');
 }
 catch(e) {
 	debuglog('Require noble module: ', e.message);
```
Comment (gvancuts) [inline ocf-servers/js-servers/environmental_sensor.js]: One suggestion from me here is to add a note in the code (and possibly in debuglog too) that we are nevertheless proceeding but that all data sent is fake data.

File: ocf-servers/js-servers/environmental_sensor.js [hunk 2]
```diff
@@ -18,17 +18,19 @@ var	device = require('iotivity-node')('server'),
...
-	process.exit(0);
+	//process.exit(0);
 }
 
+if(noble){
 // intitialize BLE, make sure BT is enabled with 'rfkill unblock bluetooth'
```
Comment (gvancuts) [inline ocf-servers/js-servers/environmental_sensor.js]: The indentation looks a little awkward now that we've added an if loop. Can we fix this?

File: ocf-servers/js-servers/environmental_sensor.js (hunk 1 above)
Comment (gvancuts) [reply ocf-servers/js-servers/environmental_sensor.js]: Actually... the way you've done it is currently how it's done in all other OCF JS servers... let's keep it as it is then. We can change that in batch mode later if we decide too...

Comment (ichbinblau) [reply ocf-servers/js-servers/environmental_sensor.js]: Yes, I will refer to the debug messages in other OCF js server.

File: ocf-servers/js-servers/environmental_sensor.js (hunk 2 above)
Comment (ichbinblau) [reply ocf-servers/js-servers/environmental_sensor.js]: Sure, I will fix it .,

**Review by ichbinblau — COMMENTED** [review]
Will add more comments, debuglog and change the indentation in if clause.

**Comment by ichbinblau** [comment]
Hi, @gvancuts . I have updated the code per your review,  The debuglog indicating "No noble module" is located in the catch clause of loading "noble" module, just like how the other OCF js server handles it.

**Comment by kiendinh** [comment]
@ichbinblau Thank you. It looks good to me. One thing is that I would like to make it clear that it's in sim mode or BLE mode. So can you change the debug message to standard output? Line #30
- debuglog('No noble module: ', e.message);
- console.log('No noble module: ' + e.message + '. Switch to simulation mode');

Also, can you change the spaces in front of lines to tab?

**Comment by ichbinblau** [comment]
Hi, @kiendinh The debug message output and the tab format in the code. Plz check again. Thanks for the review.

**Comment by kiendinh** [comment]
Thank @ichbinblau.

**Pull request merged**